src/testRunner/unittests/tsserver/getFileReferences.ts

```diff
@@ -54,5 +54,28 @@ namespace ts.projectSystem {
 
             assert.deepEqual(response, expectResponse);
         });
+

```

Comment:
We've been moving to baseline tests for FAR.  Would you mind adding one of those instead/as well?

Reply:
Do you mean to change `assert.deepEqual` to `Harness.Baseline.runBaseline`?

Reply:
`verify.baselineFindAllReferences`

Reply:
Do you mean this [`baselineFindAllReferences`](https://github.com/microsoft/TypeScript/blob/d5f5c6d61e06717613286546f568e80c4a8775b4/tests/baselines/reference/findAllRefsExportDefaultClassConstructor.baseline.jsonc#L33-L46) test? 

https://github.com/microsoft/TypeScript/blob/58bae8db69b275a3efa57b14b486778c55185552/src/harness/fourslashImpl.ts#L1152

https://github.com/microsoft/TypeScript/blob/c18791ccf165672df3b55f5bdd4a8655f33be26c/src/services/types.ts#L490

https://github.com/microsoft/TypeScript/blob/c18791ccf165672df3b55f5bdd4a8655f33be26c/src/services/types.ts#L1076-L1079

https://github.com/microsoft/TypeScript/blob/c18791ccf165672df3b55f5bdd4a8655f33be26c/src/services/types.ts#L1081-L1083

https://github.com/microsoft/TypeScript/blob/c18791ccf165672df3b55f5bdd4a8655f33be26c/src/services/types.ts#L910-L913

Reply:
Interesting.  You might need to put it in the server fourslash tests (they look the same but behave differently - in particular, they cover session.ts).  When I may a dummy version of this change (always drop), it broke a bunch of baselines - that might be an easy way to find them.

Reply:
I've checked `fourslash`/`fourslash/server`, and it seems that `baselineFindAllReferences` doesn't cover `session.ts`.

>  it broke a bunch of baselines - that might be an easy way to find them.

Baseline changes related to [/unittests/tsserver/projectReferences](https://github.com/microsoft/TypeScript/blob/main/src/testRunner/unittests/tsserver/projectReferences.ts), which uses `baselineTsserverLogs`.

Reply:
Huh, I guess I misremembered.  Sorry about that.